code/modules/power/engines/singularity/singularity.dm

```diff
@@ -21,7 +21,8 @@
 	move_resist = INFINITY	//no, you don't get to push the singulo. Not even you OP wizard gateway statues
 	var/consume_range = 0 //How many tiles out do we eat
 	var/event_chance = 15 //Prob for event each tick
-	var/target = null //its target. moves towards the target if it has one
+	var/target = null //Its target. Moves slowly towards this.
+	var/becaon_target = null //Syndicate singularity beacon. Moves fast towards this.
```

Comment:
```suggestion
	var/beacon_target = null //Syndicate singularity beacon. Moves fast towards this.
```
minor spelling thing, you'll want to change this in its usages as well

Reply:
Whoops